packages/cozy-client/src/helpers.spec.js

```diff
@@ -150,6 +151,83 @@ describe('generateWebLink', () => {
   })
 })
 
+fdescribe('deconstructWebLink', () => {
+  it.each([
+    [
+      'https://claude-notes.mycozy.cloud',
+      'flat',
+      {
+        protocol: 'https:',
+        cozyName: 'claude',
+        cozyDomain: 'mycozy.cloud',
```

Comment:
I'm not confortable with the `cozyDomain` notion. To me, in that case, the cozyDomain is : `claude.mycozy.cloud`

Reply:
Do you have any suggestion on how to name that part? Would `cozyBaseDomain` work?

Reply:
Let's go for cozyBaseDomain

Reply:
Fixed in https://github.com/cozy/cozy-client/compare/38c31b33efee7fe4485930f29698112f9be495c2..ada77f37c863df55c5e555de4960603ce5a1d48c